docs/en/connector-v2/sink/Enterprise-WeChart.md

```diff
@@ -0,0 +1,27 @@
+# Enterprise WeChart
+
+> Enterprise WeChart sink connector
+
+## Description
+
+A sink plugin which use Enterprise WeChart robot send message
+
+##  Options
```

Comment:
Missing description of other options, such as common options in http connector

Reply:
> Missing description of other options, such as common options in http connector

I have added description of other options，detail in new commit